olm_deploy/scripts/env.sh

```diff
@@ -3,7 +3,7 @@ set -eou pipefail
 
 LOGGING_VERSION=${LOGGING_VERSION:-5.6}
 LOGGING_FLUENTD_VERSION=${LOGGING_FLUENTD_VERSION:-1.14.6}
-LOGGING_VECTOR_VERSION=${LOGGING_VECTOR_VERSION:-0.21-rh}
+LOGGING_VECTOR_VERSION=${LOGGING_VECTOR_VERSION:-v0.23-rh}
```

Comment:
change this back to v0.21-rh

Reply:
without `v` to 0.21-rh, right @vimalk78 ?

Reply:
yes, without `v`